yarn.lock

```diff
@@ -12,87 +12,149 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@algolia/client-abtesting@npm:5.2.4":
-  version: 5.2.4
-  resolution: "@algolia/client-abtesting@npm:5.2.4"
+"@algolia/client-abtesting@npm:5.9.1":
```

Comment:
@raineorshine, do we have a standard approach for manually adding/updating dependencies? This PR only added `lottie-react` and `canvas`, yet the lockfile has many, many unrelated updates. I see we have `lockfile-lint` set up, but that is limited in what it can cover. It would be nice to have more narrow updates to the lockfile so that they can at least be somewhat reviewed for any possible concerns. Changes like this are also likely to lead to merge conflicts.

Reply:
I don't have a process for that, but I'm open to suggestions.

Reply:
I'll look into this, but I'm positive there is a simple solution. Before I can do that, though, I need to figure out why my own `yarn` seems to be somewhat incompatible with what this project is using. When I update dependencies myself, the lockfile heading changes from:

```
# This file is generated by running "yarn install" inside your project.
# Manual changes might be lost - proceed with caution!
```

To:

```
# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
# yarn lockfile v1
```

Along with other formatting changes causing **_every_** line to get updated in the file 😅

I'm guessing this is a conflict between the old yarn installed with `npm` and the new yarn installed via `corepack enable`. I tried switching from the old one to the new one (assuming that is the issue), but haven't had any luck getting a different result yet. I'll debug that on my own time later.

Reply:
Yes, it does sound like a yarn version mismatch. Maybe the executable is not correct in your PATH?

I'm on yarn `v4.4.1`.

Reply:
It was a journey, but I'm now on yarn `4.4.1` as well. Looks like `yarn add` typically only adds the required dependencies without doing miscellaneous updates.

@ohrytskov, I see you've cleaned up the unrelated `yarn.lock` updates when merging in the main branch with https://github.com/cybersemics/em/pull/2462/commits/936e4018eee49772a91a0cbdd8aae38ed809f106. Thank you very much for that!

Did you do anything other than `yarn add [package]` when initially adding the dependencies? I'm just trying to understand why there were so many changes previously.

Reply:
I'm also using Yarn v4.4.1 and Node.js v20.17.0.

Reply:
With corepack enabled.